### PR TITLE
Add website field

### PR DESCRIPTION
**Pull request by johnrhimawan**
Currently, the ```Internship``` does not have a website field. As a user applying for multiple internships, I want to be able to input the website name for the careers website of the company I am applying to so that I can have easier access to view the necessary details.

Users are now able to add website through the ```add``` command using the prefix ```/w```. Website names must be valid for it to be accepted by ```InTrack```.

### CODE REVIEW

File: src/main/java/seedu/intrack/model/internship/Website.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package seedu.intrack.model.internship;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.intrack.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an Internship's address in the internship tracker.
```
Comment (bigcrushes) [inline src/main/java/seedu/intrack/model/internship/Website.java:7]:
```suggestion
 * Represents an Internship's website in the internship tracker.
```
I think this line was missed when refactoring address to website.

File: src/test/java/seedu/intrack/logic/parser/AddCommandParserTest.java [hunk 2]
```diff
@@ -97,63 +97,63 @@ public void parse_compulsoryFieldMissing_failure() {
...
         assertParseFailure(parser, NAME_DESC_MSFT + POSITION_DESC_MSFT + INVALID_PHONE_DESC + EMAIL_DESC_MSFT
-                + ADDRESS_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Phone.MESSAGE_CONSTRAINTS);
+                + WEBSITE_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Phone.MESSAGE_CONSTRAINTS);
 
         // invalid email
         assertParseFailure(parser, NAME_DESC_MSFT + POSITION_DESC_MSFT + PHONE_DESC_MSFT + INVALID_EMAIL_DESC
-                + ADDRESS_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Email.MESSAGE_CONSTRAINTS);
+                + WEBSITE_DESC_MSFT + TAG_DESC_URGENT + TAG_DESC_REMOTE, Email.MESSAGE_CONSTRAINTS);
 
         // invalid address
```
Comment (bigcrushes) [inline src/test/java/seedu/intrack/logic/parser/AddCommandParserTest.java:141]:
```suggestion
        // invalid website
```
I think this documentation change was missed out.

File: src/test/java/seedu/intrack/logic/parser/EditCommandParserTest.java [hunk 3]
```diff
@@ -153,8 +153,8 @@ public void parse_oneFieldSpecified_success() {
         assertParseSuccess(parser, userInput, expectedCommand);
 
         // address
```
Comment (bigcrushes) [inline src/test/java/seedu/intrack/logic/parser/EditCommandParserTest.java:155]:
```suggestion
        // website
```
Another minor documentation change

**Review by bigcrushes — COMMENTED** [review]
Good job refactoring the address field to website! Just some minor documentation changes seem to have been missed but apart from that LGTM!

File: src/main/java/seedu/intrack/model/internship/Website.java (hunk 1 above)
Comment (johnrhimawan) [reply src/main/java/seedu/intrack/model/internship/Website.java]: Thank you for your suggestion, the changes have been implemented.

File: src/test/java/seedu/intrack/logic/parser/AddCommandParserTest.java (hunk 2 above)
Comment (johnrhimawan) [reply src/test/java/seedu/intrack/logic/parser/AddCommandParserTest.java]: Thank you for pointing that out, I have fixed the documentation.

File: src/test/java/seedu/intrack/logic/parser/EditCommandParserTest.java (hunk 3 above)
Comment (johnrhimawan) [reply src/test/java/seedu/intrack/logic/parser/EditCommandParserTest.java]: Thank you for noticing, I have made the appropriate changes.

**Comment by bigcrushes** [comment]
Thanks for making the changes and good job even making the links clickable. LGTM!

**Pull request merged**